tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/TestFunctions.py

```diff
@@ -0,0 +1,88 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+# Functions for testing CylindricalBlastWave.cpp
+def cylindrical_blast_wave_compute_piecewise(x, inner_radius, outer_radius,
+                                             inner_value, outer_value):
+    radius = np.sqrt(np.square(x[0]) + np.square(x[1]))
+    if (radius > outer_radius):
+        return outer_value
+    elif (radius < inner_radius):
+        return inner_value
+    else:
+        piecewise_scalar = (-1.0 * radius + inner_radius) * np.log(outer_value)
+        piecewise_scalar += (radius - outer_radius) * np.log(inner_value)
+        piecewise_scalar /= inner_radius - outer_radius
+        return np.exp(piecewise_scalar)
+
```

Comment:
Could you run `autopep8 -i FILENAME` on this file?

Reply:
Sure! Would you mind adding this to the docker container? As more and more python gets run, this tool will be in demand. (I added it by hand with `pip install --upgrade autopep8`)